src/systems/systems/src/pressurization/mod.rs

```diff
@@ -30,26 +58,96 @@ impl Pressurization {
         }
 
         Self {
+            cabin_pressure_simulation: CabinPressure::new(),
             cpc: [CabinPressureController::new(); 2],
-            outflow_valve: PressureValve::new(),
+            outflow_valve: PressureValve::new_outflow_valve(),
+            safety_valve: PressureValve::new_safety_valve(),
+            residual_pressure_controller: ResidualPressureController::new(),
             active_system: active,
             landing_elevation: Length::new::<foot>(0.),
+            departure_elevation: Length::new::<foot>(0.),
             sea_level_pressure: Pressure::new::<hectopascal>(1013.25),
             destination_qnh: Pressure::new::<hectopascal>(0.),
+            is_in_man_mode: false,
+            man_mode_duration: Duration::from_secs(0),
+            packs_are_on: true,
+            lgciu_gear_compressed: true,
         }
     }
 
-    pub fn update(&mut self, context: &UpdateContext, engines: [&impl EngineCorrectedN1; 2]) {
+    pub fn update(
+        &mut self,
+        context: &UpdateContext,
+        press_overhead: &PressurizationOverheadPanel,
+        engines: [&impl EngineCorrectedN1; 2],
+        lgciu: [&impl LgciuWeightOnWheels; 2],
+    ) {
+        self.cabin_pressure_simulation.update(
+            context,
+            self.outflow_valve.open_amount(),
+            self.safety_valve.open_amount(),
+            self.packs_are_on,
+            self.lgciu_gear_compressed,
```

Comment:
**question**: Is using `self.lgciu_gear_compressed` before modifying it below intentional? If so, what is the reason? Should that reason be added as a comment?

Reply:
It was, because it was used as a check to know if the aircraft had been initialized and then used it to set an "in the air" cabin pressure. This was far from ideal (see comment from @beheh above), so I've now changed this to add an `initialized` field to the `CabinPressureSimulation` type, and moved the update of the `self.lgciu_gear_compressed` before its use. 

Althought adding an extra field, I think this is a slightly more elegant solution and it seems to work well in the sim. Let me know your thoughts if you disagree!

Reply:
I think adding the extra field is good! Makes the intent clearer!